Components/Core/Spells/Equivalencies.lua

```diff
@@ -504,6 +509,16 @@ TMW.BE = {
 			-289657, -- Holy Word: Concentration
 			 290641, -- Ancestral Gift
 		},
+		ImmuneToSlows = {
```

Comment:
New section. Will add eng TL soon. Can go further with snare immunity definition, to include full immunities. Will ask you.